lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/util/XmlReader.java

```diff
@@ -30,24 +32,30 @@
 public class XmlReader {
     private static final Logger LOGGER = Logger.getLogger(XmlReader.class.getName());
 
-    public static boolean hasServerRoot(File file) {
-        if (!file.exists() || file.length() == 0) {
-            return false;
-        }
-
+    public static boolean hasServerRoot(String filePath) {
         XMLInputFactory factory = XMLInputFactory.newInstance();
         XMLEventReader reader = null;
+        File file = null;
+        
         try {
+            file = new File(new URI(filePath).getPath());
```

Comment:
just curious why we have to create a URI and get the path from that to create the file? What is the format of the filePath? I know we changed `\` to `/` in the calling method.

Reply:
I learned checking `file.exists()` with a `File` constructed with a string in the URI format returns `false`. We get URI formatting from using DOMDocument